<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswd</class>
 <widget class="QWidget" name="ChangePasswd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="userId_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userId_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="oldKey_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldKey_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="newKey_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newKey_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="newKeyAgain_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码确认：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newKeyAgain_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="err_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>160</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>原密码错误！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>270</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
